### PR TITLE
[V2 API] Migrate and clean up usage of observation existence API

### PR DESCRIPTION
**Pull request by shifucun**
Follow the new file layout pattern:

* Create data fetch function in fetch.py, with response simplification
* Remove v1/v2 observation existence REST call wrapper in datacommons.py

### CODE REVIEW

File: server/routes/shared_api/observation/existence.py [hunk 1]
```diff
@@ -16,22 +16,23 @@
...
-  entities = request.args.getlist('entities')
-  if not entities:
-    return 'error: must provide entities field', 400
-  variables = request.args.getlist('variables')
-  if not variables:
-    return 'error: must provide variables field', 400
-  return json.dumps(dc.observation_existence(variables, entities))
+  variables = request.json.get('variables', [])
+  entities = request.json.get('entities', [])
+  return Response(json.dumps(fetch.observation_existene(variables, entities)),
```
Comment (chejennifer) [inline server/routes/shared_api/observation/existence.py:36]: fetch.observation_existence?

maybe should add tests for these endpoints

**Review by chejennifer — COMMENTED** [review]
nice!

**Comment by shifucun** [comment]
Thanks for the review!

File: server/lib/nl/utils.py [hunk 2]
```diff
@@ -113,7 +113,7 @@ def sv_existence_for_places(places: List[str], svs: List[str],
   existing_svs = []
   for sv in svs:
     exists = False
-    for _, exist_bit in sv_existence['variable'][sv]['entity'].items():
+    for _, exist_bit in sv_existence.get(sv, {}).items():
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply server/lib/nl/utils.py:116]: ahh, yes, you're right, my bad

**Review by chejennifer — APPROVED** [review]

**Pull request merged**